From bbcf90dbe7a26bca27fbfd5ae189fe3a40180b08 Mon Sep 17 00:00:00 2001
From: Rui Matos <tiagomatos@gmail.com>
Date: Fri, 6 Dec 2013 15:31:41 +0100
Subject: [PATCH 01/12] xkb: Blacklist grp and grp_led XKB options

grp is handled in gnome-control-center and grp_led is unsupportable
with the way we handle keyboard layouts in GNOME.
---
 gtweak/tweaks/tweak_group_xkb.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gtweak/tweaks/tweak_group_xkb.py b/gtweak/tweaks/tweak_group_xkb.py
index 187eaa0..0e1c902 100644
--- a/gtweak/tweaks/tweak_group_xkb.py
+++ b/gtweak/tweaks/tweak_group_xkb.py
@@ -101,8 +101,8 @@ class TypingTweakGroup(Gtk.Box, TweakGroup):
 
     XKB_GSETTINGS_SCHEMA = "org.gnome.desktop.input-sources"
     XKB_GSETTINGS_NAME = "xkb-options"
-    # These are configurable in gnome-control-center
-    XKB_OPTIONS_BLACKLIST = {"lv3","Compose key"}
+    # These are configurable in gnome-control-center. grp_led is unsupported
+    XKB_OPTIONS_BLACKLIST = {"lv3","Compose key","grp","grp_led"}
 
     def __init__(self):
         Gtk.Box.__init__(self, orientation=Gtk.Orientation.VERTICAL, spacing=3)
-- 
2.3.6

